sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/AuthenticationRecord.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.identity;
+package com.azure.identity.implementation;
```

Comment:
why moving this to implementation instead of changing the modifier to be package-private?

Reply:
to ensure, the classes `MsalAuthenticationAccount` and `IdentityClientOptions` can access this class and the underlying implementation retains as it is.